packages/ua-devtools/src/oapp/config.ts

```diff
@@ -57,15 +58,12 @@ export const configureOAppDelegates: OAppConfigurator = withOAppLogger(
     }
 )
 
-export const configureOAppPeers: OAppConfigurator = async (graph, createSdk) => {
-    const logger = createModuleLogger('OApp')
-
-    return flattenTransactions(
```

Comment:
Unsure if I still need to `flattenTransactions` if using `createConfigureEdges`...will double check stargate repo to see how it done there